src/powershell/install.sh

```diff
@@ -207,15 +207,15 @@ install_using_github() {
     pwsh_url="https://github.com/PowerShell/PowerShell"
     find_version_from_git_tags POWERSHELL_VERSION $pwsh_url
     install_pwsh "${POWERSHELL_VERSION}"
-    if grep -q "Not Found" "${powershell_filename}"; then 
+    if grep -q "Not Found" "${powershell_filename}"; then
         install_prev_pwsh $pwsh_url
     fi
 
     # Ugly - but only way to get sha256 is to parse release HTML. Remove newlines and tags, then look for filename followed by 64 hex characters.
     curl -sSL -o "release.html" "https://github.com/PowerShell/PowerShell/releases/tag/v${POWERSHELL_VERSION}"
     powershell_archive_sha256="$(cat release.html | tr '\n' ' ' | sed 's|<[^>]*>||g' | grep -oP "${powershell_filename}\s+\K[0-9a-fA-F]{64}" || echo '')"
     if [ -z "${powershell_archive_sha256}" ]; then
-        echo "(!) WARNING: Failed to retrieve SHA256 for archive. Skipping validaiton."
+        echo "(!) WARNING: Failed to retrieve SHA256 for archive. Skipping validation."
```

Comment:
Can you bump the patch version for this Feature? Thanks!

Reply:
Done